### PR TITLE
Introducing: /vg/ui! 🎉

### PR DESCRIPTION
**Pull request by nervere**
## What is /vg/ui?
I literally just renamed TGUI.

## Why did you do that?
It's a powerful, robust UI system that some people endlessly complain about because of the name. Changing the name is trivial, so I did it. I hope people will actually use this instead of the nanoUI garbage that breaks every 5 minutes.

## Licensing?
I don't know if this is even a concern, but all of TGUI is MIT which is GPL-compatible anyway.
Not that it matters since it's already ported as of three years ago...

## Did you test this?
Yes, I tested the syndicate uplink, cargo merch computer, MSGSU, GPS, security camera console, PCMC, power monitor, robotics console, and mecha console. That's everything that uses VGUI. No issues.

## That's a lot of files changed. I'm not reading all that! What did you *actually* do?
All variables, documentation, filenames, directories, etc. have been changing from TGUI to VGUI. No, I'm not putting in slashes, that's gonna break everything. No other changes have been made and it functions exactly the same otherwise.

## Maintainability
Is this a change that's maintainable? Well, that's up to collabs, but consider the following:
Our version of TGUI, added three years ago: https://github.com/vgstation-coders/vgstation13/pull/27012
...has only received two updates in all that time, latest 18 months ago: https://github.com/vgstation-coders/vgstation13/pull/32101
(this is obviously not counting the automatic depend-a-bot PR's for TGUI dependencies, those are automatic)

In short, our version of TGUI is already severely out of date, and the guy who was maintaining it (Damian) no longer codes for the project. The potential benefit (more people using it) outweighs the potential cost (having to find and replace TGUI with VGUI in future updates that probably aren't ever coming). If we were a downstream or a fork, I'd get the maintainability argument, but we're not and never will be.

[system]

### CODE REVIEW

**Comment by lol-and** [comment]
I don't know what this does and what it accomplishes but it has TG written somewhere in it so thumbs down

**Comment by wotisjanitor** [comment]
I have literally been asking for this for years. /vg/ rising?

**Comment by Inorien** [comment]
\>VGUI

what the fucking fuck is VG

**Comment by nervere** [comment]
It's the closest you're gonna get without messing up the code by including slashes!

**Comment by SonixApache** [comment]
i don't see a single purpose for this

**Comment by Pastamadness** [comment]
Meme PR

**Comment by GreatBigFailure** [comment]
Imagine using something that breaks every 5 minutes because people you hate told you to dislike something you normally wouldn't care about.

**Comment by nervere** [comment]
What does this comment mean? The only thing that regularly breaks is nanoUI! TGUI is leagues ahead in reliability.

**Comment by Eneocho** [comment]
I do not play on TG, do they have the issue where TGUI is sometimes blank for some ungodly reason or is that fixed there.

**Comment by lol-and** [comment]
I thought that had something to do with IE or something. It hasn't happened to me in a while

**Comment by Kurfursten** [comment]
I have no idea why this PR is controversial. We've had tgui for 2 years and no one voted against it at all back then. If you had named this something like "updates our tgui port to the latest" probably like one person would have even opened the PR.

To be clear, you haven't moved anything to tgui that wasn't already on it?

**Comment by Exxion** [comment]
It's controversial because it's pointless. Anyone saying not to use TGUI simply because it has TG in the name is not worth listening to.

**Comment by GreatBigFailure** [comment]
> What does this comment mean? The only thing that regularly breaks is nanoUI! TGUI is leagues ahead in reliability.

That's what I'm saying. The idea of avoiding TGui because something something I was told by the people I hate on my korean paddleboat forum that it was bad, is very silly. I appreciate the intended compromise here but I don't think that kind of smooth brain nonsense should be compromised with.

**Comment by wotisjanitor** [comment]
Look at how militant they have gotten. Look how they recoil and show themselves when exposed to the light. We are reforging our identity. WE will not let the soulless minions of orthodoxy do what they have done to other servers. No one is coming to help us. The victory is only ours to snatch from the jaws of defeat!

**Comment by toomykins** [comment]
can we make it /vg/UI instead
also I don't see anyone ever working on this clusterfuck in the future. No reason to up- or down-vote for me.

**Comment by nervere** [comment]
> To be clear, you haven't moved anything to tgui that wasn't already on it?

Correct. I have not added or removed TGUI functionality from anything. This is a rename only.

**Comment by deadc4fe** [comment]
as a victim of the exact complaining this PR would hope to quell, and having dropped my dual-boot setup for pure ganoo/linux, I think any effort spent on joke PRs would be better used looking into polyfills for the existing TGUI framework, changing the framework to something IE7 compatible (would probably not require changing any backend code), or just waiting for WebView2 support in Byond (nobody really cares about changing UIs for now anyway, it seems)
for however polished TGUI looks, ultimately it is less reliable than the all server-side UIs that make up the majority of /vg/'s interfaces

**Comment by nervere** [comment]
A large part of my motivation to make this PR (I've been thinking about it for a while) was based on the experience you went through in that PR. It was really painful to see so much work go into implementing TGUI only for it to go to waste. Now, I will not dishonestly claim that the only qualms people had with it were the name of the UI, but it was certainly too much of a factor to be comfortable with. With this, I hoped to prevent that from happening again.

That said, I find your argument for alternatives compelling, and upon reflection I may have over-valued the potential usefulness of the UI. This PR isn't very productive by its very nature, and efforts would be better spent elsewhere. For that reason, in addition to some people coming forward to let me know about Linux compatibility issues being worse than initially realized, I'll close this.

I appreciate the discussions had in this thread.

**Comment by wotisjanitor** [comment]
Remember this moment. Remember what we did here
